lib/Doctrine/ORM/Internal/CommitOrderCalculator.php

```diff
@@ -149,11 +149,15 @@ public function sort()
      *
      * @param \stdClass $vertex
      */
-    private function visit($vertex)
+    private function visit($vertex, $minimumWeight = 0)
```

Comment:
This branch requires 7.2, you can use a type hint:

```suggestion
    private function visit($vertex, int $minimumWeight = 0)
```

Reply:
I applied the suggested change in the last (squashed) commit. Thank you.